hashes/src/hash160.rs

```diff
@@ -57,16 +57,16 @@ mod tests {
         #[derive(Clone)]
         #[cfg(feature = "alloc")]
         struct Test {
-            input: Vec<u8>,
-            output: Vec<u8>,
+            input: [u8; 65],
+            output: [u8; 20],
             output_str: &'static str,
         }
```

Comment:
Not dependent on `alloc` anymore, `#[cfg(feature = "alloc")]` may be removed. I think there are more of these cases in this PR.

Reply:
Maybe thats the thing for another PR?